resources/dev_resources/EndpointResourcesList.json

```diff
@@ -13,5 +13,15 @@
         "EndpointName":"Lantern",
         "FileName":"LanternEndpointSources.json",
         "URL": ""
+    },
+    {
+        "EndpointName":"CareEvolution",
```

Comment:
I now see that you're using the EndpointName in the query-endpoint-resources script. I think the EndpointName is supposed to be more of like a format type, so like EndpointName here would be "Lantern" since they both use the Lantern parsing format. But then you can't use it in the below script... maybe we should add another field, EndpointName can stay the same but then add a "FormatType" field that references the correct format type for the list which could then be used in the fetcher?

Reply:
I guess alternatively we could use the url in the below script but that feels bad lol